version.properties

```diff
@@ -1,2 +1,2 @@
-version=2.2.0-beta.2
+version=2.2.0
```

Comment:
I removed central only temporarily. After the release, I have 24 hours to validate the binary in Bintray. During this time I can delete it if needed. I intend to sync to central manually from BinTray Web interface once I smoke test the release. After few successful releases, I will put back automatic maven central.